sljit_src/sljitNativeARM_T2_32.c

```diff
@@ -509,6 +509,9 @@ SLJIT_API_FUNC_ATTRIBUTE void* sljit_generate_code(struct sljit_compiler *compil
 SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_has_cpu_feature(sljit_s32 feature_type)
 {
 	switch (feature_type) {
+	case SLJIT_HAS_CTZ:
```

Comment:
This should be armv6, and should be somewhere at the end.

Reply:
Since the end has a `default: return 0`, thought it was more appropriate for it to be at the top to help distinguish it.

About it being armv6, I have to admit I was surprised to find out that `CTZ` was returning 1 originally, after all it is not a direct translation and requires at least 2 instructions when `RBIT` is available; wouldn't that count as "emulation"?

Reply:
Well, 2 should means: emulation is costly, so if you can avoid this instruction by a different code path, avoid it. The two instruction sequence is simple enough.

However, I am open to use a better description system, maybe use define constants instead of 0-2 return values. If you have time, patches are welcome!

Reply:
one way to do that is to return the number of native instructions that are required by emulation so the cost can be also communicated back to the callers in an objective way instead of being implicit.

Reply:
I am not sure that helps too much from developer perspective. I prefer something which refers to complexity. The question is: am I recommended to use this, or should avoid it when possible.